### PR TITLE
icu: add recipe for icu 63.2

### PR DESCRIPTION
**Pull request by kenmays**
**Unicode 11 for Haiku**

Tested on hrev54302 x86_64.
Passed all tests successfully.

### CODE REVIEW

File: dev-libs/icu/icu-62.2.recipe [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+	# Created during build and prevent rebuilds.
+	#rm -rf tools/tzcode/vanguard
+	#cp ../../../sources-3/tzdata*.tar.gz tools/tzcode/
+
+	unzip -oq ../../../sources-2/icu4c-*-data.zip
+	rm -f data/in/icudt??l.dat
+	autoconf
+	runConfigure ./configure \
+		--disable-samples --disable-extras --with-data-packaging=archive \
+		--disable-renaming --disable-layoutex
```
Comment (extrowerk) [inline dev-libs/icu/icu-62.2.recipe:139]: `--disable-renaming --disable-layoutex` seems to be a new addition to the recipe. Would be nice to know what they do and why they required.

Comment (kenmays) [reply dev-libs/icu/icu-62.2.recipe]: Those are optional.

**Comment by pulkomandy** [comment]
Why add old versions of ICU?

**Comment by kenmays** [comment]
I verified ICU 63.2 works fine so we can update Webkit. ICU 66/67.1 had some testing issues on Haiku that I'm researching. Hopefully, you've had much better luck.

**Comment by pulkomandy** [comment]
Given the work required to update haiku to use a newer ICU (recipes for haikuportscross, updating the build repos, etc) I'd rather update directly to the latest version.

**Pull request closed without merging**